src/Microsoft.DocAsCode.Build.RestApi/RestApiDocumentProcessor.cs

```diff
@@ -141,11 +142,11 @@ private static bool IsSwaggerFile(string filePath)
         {
             try
             {
-                var dictionary = JsonUtility.Deserialize<Dictionary<string, object>>(filePath);
-                object swaggerValue;
-                if (dictionary.TryGetValue("swagger", out swaggerValue))
+                var jObject = JObject.Parse(File.ReadAllText(filePath));
```

Comment:
yes it's better.

---
In reply to: [74371215](https://github.com/dotnet/docfx/pull/599#discussion_r74371215) [](ancestors = 74371215)